SSPS/src/SSPS.jl

```diff
@@ -175,6 +173,9 @@ function perform_inference(timeseries_filename::String,
     if proposal == "uniform"
         update_loop_fn = uniform_update_loop
     end
+    if multithread
+        update_loop_fn = multithread_update_loop
```

Comment:
If I'm reading correctly, specifying `multithread` forces `proposal=sparse` and will override `proposal=uniform`.  That would be nice to document somewhere, probably in the help message.

Reply:
You are reading correctly. The uniform proposal is only interesting as a baseline of comparison and isn't actually useful. So the correct course of action is probably to deprecate the uniform proposal and eventually remove it.